moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -292,6 +296,11 @@ const ProjectsListViewTable = ({ title, query, searchTerm, referenceData }) => {
           {entry.project_name}
         </RouterLink>
       ),
+      cellStyle: {
+        position: "sticky",
+        left: 0,
+        backgroundColor: "white",
+      }
```

Comment:
just saw the build error related to a duplicate entry for `cellStyle` - you need need to get the whitespace/nowrap in here.